src/raps/execute.ts

```diff
@@ -139,11 +145,11 @@ const waitForNodeAck = async (hash: string, provider: Signer['provider'], tries
 export const walletExecuteRap = async (
   wallet: Signer,
   type: RapTypes,
-  parameters: RapSwapActionParameters<'swap' | 'crosschainSwap' | 'claimBridge'>
+  parameters: RapSwapActionParameters<'swap' | 'crosschainSwap' | 'claimBridge' | 'claimClaimable'>
 ): Promise<{ nonce: number | undefined; errorMessage: string | null }> => {
   // NOTE: We don't care to track claimBridge raps
   const rap =
-    type === 'claimBridge'
+    type === 'claimBridge' || type === 'claimClaimable'
```

Comment:
#nit, but can you define this as an array outside of the function so we can grow this later on if we need to without a bunch of conditionals existing here?